### PR TITLE
fix append and add remove

### PR DESCRIPTION
**Pull request by tyler-hoffman**

### CODE REVIEW

File: lib/jqlite.js [hunk 1]
```diff
@@ -49,9 +49,16 @@
...
           newElem = document.createElement(newElem);
         }
-        this.element.appendChild(newElem);
-        return new JQLite(newElem);
+        this.element[0].appendChild(newElem);
+        return new JQLite([newElem]);
       }
+
+      JQLite.prototype.remove = function() {
+        this.element.forEach(function(element) {
```
Comment (pegurnee) [inline lib/jqlite.js:57]: is remove supposed to be a terminal function?

Comment (tyler-hoffman) [inline lib/jqlite.js:57]: Oh, good point. it should probably return the element removed?

Comment (pegurnee) [inline lib/jqlite.js:57]: only if it isn't supposed to be terminal. Honestly, returning the parent might be more useful, but returning the removed element might be more predictable.

Comment (tyler-hoffman) [inline lib/jqlite.js:57]: I think we should err on the side of predictability. So non-terminal, and returning the removed element. It's easy enough just to call it's parent function from there.

Comment (tyler-hoffman) [inline lib/jqlite.js:57]: I added the change.

**Pull request merged**